comp/swerve100/src/main/java/org/team100/frc2024/LEDSubsystem.java

```diff
@@ -45,29 +48,30 @@ public LEDSubsystem(
 
     @Override
     public void periodic() {
+        m_indicator.setStripSolid(LEDStrip.strip.BACKLIGHT, State.WHITE);
```

Comment:
there's no reason to keep resetting the white side to white if it never changes.  is it never going to change?